### PR TITLE
Added properties to the create-cloud-image job.

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

**Comment by elopio** [comment]
Requires ubuntu-core/snappy-cloud/image/pull/45.

File: containers/jenkins-master/config/jobs/create-cloud-image-all-snaps/config.xml [hunk 1]
```diff
@@ -45,13 +50,31 @@
...
+    i386)
+      properties="architecture='i686'"
+      ;;
+    arm64)
+      properties="architecture='aarch64'"
+      ;;
+    ppc64el)
+      properties="architecture='ppc64le'"
+      ;;
+  esac
```
Comment (fgimenez) [inline containers/jenkins-master/config/jobs/create-cloud-image-all-snaps/config.xml]: It would be nice to add a default value, the param can come empty

**Comment by fgimenez** [comment]
Nice, only a minor concern :+1:

File: containers/jenkins-master/config/jobs/create-cloud-image-all-snaps/config.xml (hunk 1 above)
Comment (elopio) [inline containers/jenkins-master/config/jobs/create-cloud-image-all-snaps/config.xml]: added.

**Pull request merged**